api/src/pcapi/core/users/external/zendesk.py

```diff
@@ -0,0 +1,266 @@
+"""
+Update user profile attributes in Zendesk to help support team check information quickly.
+This module is called when webhook receives a ticket update notification.
+
+Zendesk subscription does not include access to sandbox for testing and staging.
+So ZENDESK_API_URL will be empty for non-production platforms, except when working with a free evaluation account which
+is limited to 14 days after creation.
+"""
+import base64
+from datetime import datetime
+import json
+import logging
+from typing import Iterable
+from typing import Optional
+from typing import Union
+from urllib.parse import quote
+
+from markupsafe import Markup
+import requests
```

Comment:
Mieux vaut importer notre wrapper qui logge, ajoute un timeout, etc. : 

```suggestion
from pcapi.utils import requests
```

Son API est la même.